test/unit/specs/http_connector.js

```diff
@@ -460,7 +460,7 @@ describe('Http Connector', function () {
       var con = new HttpConnection(new Host({ suggestCompression: true }));
       var respBody = 'i should be encoded';
       var server = nock('http://localhost:9200')
-      .matchHeader('accept-encoding', 'gzip,deflate')
+      .matchHeader('Accept-Encoding', 'gzip,deflate')
```

Comment:
Changing case didn't help or anything, but it seemed like a good thing anyway.

Reply:
Pretty sure node normalizes the headers to be lower-case before exposing them on `req/resp.headers`, which is why I generally use lower-case, but 🤷‍♂️

Reply:
Okay. thanks, I'll change it back. it doesn't matter and it's not a needed change.

Reply:
Actually I'm going to leave it, the other test has it in the same hyphenated caps casing.